lib/routes/callback/index.js

```diff
@@ -5,6 +5,44 @@ const { TooManyRequests } = require('http-errors')
 const { metricsInsert } = require('../metrics/query')
 const { registerUpdate } = require('./query')
 
+/**
+ * Allows users to request a callback from a Contact Tracer. If accepted, the request is added to the
+ * AWS callback queue with information about the user to call. The user's phone # is submitted as
+ * part of this request, but the phone # IS NOT stored with the user's database record. It is included
+ * in the body of the AWS callback queue message so that Contact Tracers know which number to call.
+ *
+ * Note that within the AWS callback queue the user's ID and phone # are stored together. This is a
+ * potential PII issue as it provides an identifiable mapping between users and their phone numbers.
```

Comment:
Is this an actual issue? Given the limited data stored elsewhere what is personal beyond the mobile number?

Reply:
I think the potential for a PII/GDPR issue exists and as such should be documented. As currently implemented, this repository loses control over what happens with the User ID and phone# once they're added to the AWS queue.

The latest commit rephrases this entire paragraph to hopefully clarify the issue.

Reply:
I do have a potential solution that closes this PII/GDPR issue completely (and allows for additional features like cancelling callbacks and updating phone #s). I want to give it a little more thought before I document it, but do want to share that I've got something brewing.

Reply:
I've entered #20 to discuss a potential solution.